client/signup/jetpack-connect/plans-landing.jsx

```diff
@@ -58,7 +56,7 @@ const PlansLanding = React.createClass( {
 			</div>
 		);
 	}
-} );
+}
 
 export default connect(
 	() => {
```

Comment:
I'd prefer to get this in a PR. I'm trying to limit the scope, I've had several PRs really get away from me in terms of size. There are a few fixes that can be refactored here, but if you don't think it's a blocker, I'd rather leave it out of this.